metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -106,6 +111,8 @@ object MetalsServerConfig {
           openFilesOnRenames = true,
           executeClientCommand = ExecuteClientCommandConfig.on,
           globSyntax = GlobSyntaxConfig.vscode,
+          isInputBoxEnabled = true,
```

Comment:
I had to change it as well, because entering name for the new file cannot be expressed via `showMessageRequest`. This resulted into a bit changed UI when formatting and no scalafmt version is set.
Also, I would expect for such configs to be set by client, but okay, I mimicked it with `isPickInputEnabled`.

Reply:
I would even remove it from here, since we want to drop the properties.